Fix indentation

georgewitteman opened the pull request:
This fixes an issue where any `end` statement was getting unindented to the beginning of the line.

![2019-06-03 23 10 02](https://user-images.githubusercontent.com/5918935/58930025-a8098980-870e-11e9-8771-0a93e9447f67.gif)

krobelus committed:
use standard definition for iskeyword, as for other shells

This effectively removes "-", "." and "/" from the set of word characters.

Prior to this, a `w` would skip over the entirety of /some/long/path-name. This
is probably not what users expect, since they can use `W` for that.

georgewitteman committed:
Fix indentation

georgewitteman committed:
Fix indentation for case and else statements

georgewitteman committed:
Fix indentation for switch statements

sudoforge commented:
Leaving a comment here for reference (and to easily see the status of this PR in the following issue): https://github.com/sheerun/vim-polyglot/pull/408

gpanders committed:
Filter blank completions

georgewitteman committed:
Merge pull request #1 from gpanders/master

Filter blank completions

georgewitteman committed:
Merge pull request #2 from krobelus/default-iskeyword

use standard definition for iskeyword, as for other shells

mk12 committed:
Rewrite fish#Indent()

Fish indentation now behaves correctly in many more cases. It has also been
tested with vim-endwise.

georgewitteman committed:
Merge pull request #3 from mk12/fix-indent

Rewrite fish#Indent()
